### ISSUE TITLE
about us page!

### ISSUE THREAD

**Issue by akshay1027** [DhairyaBahl/React-Messenger-App#132]
Add about us page in the application!

**Comment by DhairyaBahl** [comment]
@akshay1027 What exactly do you want to add in the about us page ? Kindly explain the issue.

**Comment by akshay1027** [comment]
We can add why we created this website, contact links, link to GitHub source code and stuff like that, is what I thought, let me know what you think @DhairyaBahl

**Comment by DhairyaBahl** [comment]
@akshay1027 Okay ! Got your point. We can definitely do this ! We can also add some contributing guidelines or that kind of stuff over there. and we can use [ i ] icon in the header which will take us to that page or simple about us will also do the task.

But problem is , day by day the app is getting quite messy due to features that are being added. so do ask me about the position for that button / hyperlink. 

All the best

**Comment by akshay1027** [comment]
can you please tell me where we create the place the about us link in navbar! I also thought we can maybe make it as a separate website itself in a different folder in the same repo so that the app is light, maybe it won't make much off a difference but i thought it would be the best option. let me know your opinion! thank you so much! @abhijeet007rocks8 @DhairyaBahl

**Comment by DhairyaBahl** [comment]
@akshay1027 Yes we can create a folder for the same in this repo without any issues. @abhijeet007rocks8 Kindly help him out with the location for about us page in the navbar ! 

Although I would highly recommend that we should make a menu bar under the header which will be enabled with a toggle button in the header ! And all the features that are currently present in the header will then be sent in that menubar hence making the header less messy. Hope you are getting my point.

Thanks and regards ^_^

**Comment by akshay1027** [comment]
Yes so I will divide these tasks into few separate tasks as this is a big task all as a whole! First I will try to make the responsive navbar with toggle @DhairyaBahl @abhijeet007rocks8

**Comment by abhijeet007rocks8** [comment]
@akshay1027 @DhairyaBahl  What I would suggest is to make a hamburger menu for both desktop and mobile version which contains options for all the additional feature like theme changer about us page and can add future features there too.

**Comment by akshay1027** [comment]
Okay so first I will do this 🙌🏻

**Comment by akshay1027** [comment]
How I am planning to work:
1) hamburger navbar
2) about us new project setup, simple layout, content
3) Footer for this 
4) contact us form
5) backend for the same 
6) showcasing contributors in about us page 

All these I have in mind now, so I will make seperate PR for them as every task is a level2 or level3 on its own and will take time. I will start with hamburger menu

**Comment by abhijeet007rocks8** [comment]
cannot we create a separate folder for about us and use router to go to that page from main page

**Comment by akshay1027** [comment]
Yea we definitely can but are you guys okay with that ? I thought we will use express js mongo db as backend, anything is fine for me @abhijeet007rocks8. Let me know what you think

**Comment by akshay1027** [comment]
> How I am planning to work:
> 
> 1. hamburger navbar
> 2. about us new project setup, simple layout, content
> 3. Footer for this
> 4. contact us form
> 5. backend for the same
> 6. showcasing contributors in about us page
> 
> All these I have in mind now, so I will make seperate PR for them as every task is a level2 or level3 on its own and will take time. I will start with hamburger menu

We will also add an FAQ section on the about us page? @DhairyaBahl @abhijeet007rocks8, I think it would be nice.

**Comment by akshay1027** [comment]
@abhijeet007rocks8 for contact us form, how do you guys want me to do the backend? I think it would be best if we use email.js, what say?

**Comment by abhijeet007rocks8** [comment]
@akshay1027 is the issue complete

**Comment by akshay1027** [comment]
Contributors page and FAQ page is there for me to do @abhijeet007rocks8
